packages/web-components/fast-components/src/picker/picker.styles.ts

```diff
@@ -0,0 +1,52 @@
+import { css, ElementStyles } from "@microsoft/fast-element";
+import {
+    ElementDefinitionContext,
+    FoundationElementDefinition,
+} from "@microsoft/fast-foundation";
+import { designUnit, neutralLayerFloating } from "../design-tokens";
+import { heightNumber } from "../styles/index";
+
+export const pickerStyles: (
+    context: ElementDefinitionContext,
+    definition: FoundationElementDefinition
+) => ElementStyles = (
+    context: ElementDefinitionContext,
+    definition: FoundationElementDefinition
+) =>
+    css`
+        .region {
+            z-index: 1000;
+            overflow: hidden;
+            display: flex;
+        }
```

Comment:
```suggestion
        .region {
            z-index: 1000;
            overflow: hidden;
            display: flex;
            font-family: ${bodyFont};
            font-size: ${typeRampBaseFontSize};
        }
```

Adding font styling here so the region text matches the bodyFont